apps/crn-server/src/handlers/teams/sync-active-campaign-team-member-status.ts

```diff
@@ -0,0 +1,101 @@
+/* istanbul ignore file */
```

Comment:
i think there is far too much logic to just ignore this file, in fact I don't think we should ever ignore any handlers unless maybe there are using some modules which fire immediate side effects and then we should make sure to only do basic DI and setup in there and move all the logic elsewhere where it is properly tested. Here this is not the case